sips/remove_decided_msg.md

```diff
@@ -0,0 +1,67 @@
+| Author      | Title                   | Category | Status              | Date       |
+|-------------|-------------------------|----------|---------------------|------------|
+| Alan Li     | Remove decided message  | Core     | open-for-discussion | 2025-05-23 |
+
+## Summary  
+In QBFT consensus, decided message is a protocol message that consists of at least a quorum of valid commit messages. Operators can instantly terminate consensus if a valid decided message is received. However, it is an optional message to help nodes to terminate consensus. Operators are able to terminate consensus as long as a quorum of valid commit messages is received. This SIP proposes to remove the decided message from the consensus protocol.
+
+## Motivation
+Since it is not a required message, removing the decided message will reduce the processing overhead to an operator. Currently, decided messages accounted for 6.3% of the bandwidth and 3.7% of the messages in the network. Removing the decided message will reduce the bandwidth and the messages by 6.3% and 3.7% respectively.
+
```

Comment:
maybe it is a good idea to explain why it isn't required:
1. quorum of post-consensus acts like a decided message
2. The newer version of the explorer doesn't rely on it